crates/common/consensus/src/fork_choice/constants.rs

```diff
@@ -65,6 +70,9 @@ pub const SLOTS_PER_HISTORICAL_ROOT: u64 = 8192;
 pub const SYNC_COMMITTEE_SIZE: u64 = 512;
 pub const SYNC_REWARD_WEIGHT: u64 = 2;
 pub const TARGET_COMMITTEE_SIZE: u64 = 128;
+pub const TERMINAL_BLOCK_HASH: B256 = B256::ZERO;
+pub const TERMINAL_BLOCK_HASH_ACTIVATION_EPOCH: u64 = FAR_FUTURE_EPOCH;
+pub const TERMINAL_TOTAL_DIFFICULTY: u128 = 58750000000000000000000;
```

Comment:
```suggestion
```

These aren't used anymore, and are from the Merge transition I believe so we can remove them